app/Need.php

```diff
@@ -12,6 +12,6 @@ class Need extends Model
      * @var array
      */
     protected $fillable = [
-        'name', 'telephone', 'address', 'city', 'needs', 'heads'
+        'name', 'telephone', 'address','latlon', 'city', 'needs', 'heads'
```

Comment:
IMO `latlon` is a bit unreadable at first sight. Maybe something like `geolocation`?